plugin/plugins/mijia/__init__.py

```
@@ -0,0 +1,1205 @@
+        """保存凭据到文件,权限600"""
+        if not self.credential_path:
+            self.credential_path = self.data_path("credential.json")
+        self.credential_path.parent.mkdir(parents=True, exist_ok=True)
+        self.credential_path.write_text(credential.model_dump_json())
+        # 设置文件权限（仅所有者可读写）
+        import sys
+        if sys.platform == "win32":
+            try:
+                import subprocess
+                username = subprocess.check_output(
+                    ["cmd", "/c", "echo", "%USERNAME%"], text=True
```

Ruff (0.15.7)

[error] Starting a process with a partial executable path

(S607)